backend/core/query_router.py

```
@@ -0,0 +1,157 @@
+import logging
+from typing import List, Dict, Any, Optional, Tuple
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import

(F401)